src/aura/recordTypeSelection/recordTypeSelectionController.js

```diff
@@ -0,0 +1,15 @@
+({
+onSelectChange : function(component, event, helper) {
+    var selected = component.find("types").get("v.value");
+    component.set("v.selection", selected);
+
+},
+   fireApplicationEvent : function(cmp, event) {
+        var selected = cmp.find("types").get("v.value");
+        var appEvent = $A.get("e.c:compensationTypeSelected");
```

Comment:
any reason why this event is an Application event and not a Component level?

Reply:
It's a limitation on how I implemented it. The component that fires the event is in the same level as the one that receives it. If I would have had a component hierarchy I could have used a Component event.

Reply:
Mmh.. I see. it's fine for now then.